java/core/src/test/org/apache/orc/impl/TestSchemaEvolution.java

```diff
@@ -1732,10 +1732,10 @@ public void testTypeConversion() throws IOException {
         dataReader, OrcFile.WriterVersion.ORC_14, true, Integer.MAX_VALUE);
     boolean[] columns = new boolean[]{true, true, true};
     planner.parseStripe(dataReader.getStripe(0), columns)
-        .readData(null, null, false);
-    reader.startStripe(planner);
+        .readData(null, null, false, TypeReader.ReadLevel.ALL);
+    reader.startStripe(planner, TypeReader.ReadLevel.ALL);
     VectorizedRowBatch batch = readType.createRowBatch();
-    reader.nextBatch(batch, 10);
+    reader.nextBatch(batch, 10, TypeReader.ReadLevel.ALL);
```

Comment:
Use overloaded method for all and create a separate test for partial?

Reply:
To be clear you are asking can we add overloaded methods on the reader that if the level is not passed it will be treated as ALL. Please confirm.

I think that method will only be used from test, in all other cases the reader level is always supplied by the RecordReader.

Reply:
If its only used by the test then lets leave it as